examples/register_gamepads.rs

```diff
@@ -0,0 +1,98 @@
+//! Demonstrates how to register gamepads in local multiplayer fashion
+
+use bevy::{prelude::*, utils::HashMap};
+use leafwing_input_manager::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugins(InputManagerPlugin::<Action>::default())
+        .init_resource::<JoinedPlayers>()
+        .add_systems(Update, (join, jump, disconnect))
+        .run()
+}
+
+#[derive(Actionlike, PartialEq, Eq, Clone, Copy, Hash, Debug, Reflect)]
+enum Action {
+    Jump,
+    Disconnect,
+}
+
+// This is used to check if a player already exists and which entity to disconnect
+#[derive(Resource, Default)]
+struct JoinedPlayers(pub HashMap<Gamepad, Entity>);
+
+#[derive(Component)]
+struct Player {
+    // This gamepad is used to index each player
+    gamepad: Gamepad,
+}
+
+fn join(
+    mut commands: Commands,
+    mut joined_players: ResMut<JoinedPlayers>,
+    gamepads: Res<Gamepads>,
+    button_inputs: Res<Input<GamepadButton>>,
+) {
+    for gamepad in gamepads.iter() {
+        // Join the game when both bumpers (L+R) on the controller are pressed
+        // We drop down the Bevy's input to get the input from each gamepad
+        if button_inputs.just_pressed(GamepadButton::new(gamepad, GamepadButtonType::LeftTrigger))
```

Comment:
This is kinda unreliable in terms of if is works or not. Sometimes I had to press (L+R) several times before it joined. Using `pressed()` works, but it fills the terminal with "Player 0 already joined!" for a second, and it turns the behavior of "join when L and R are pressed together" to "join when L and R are both pressed, regardless of if it was done together". If this behavior is acceptable, I can make the change and could remove the print for a player joining when they are already joined.

Reply:
We should swap these both to `pressed`, not `just_pressed`. That's both more reasonable (why do we need a frame-perfect input to join the game?) and fix the reliability.